views/stats.html

```diff
@@ -62,15 +62,15 @@ <h1><span>Network Stats</span></h1>
 					<div class="row">
 						<div class="col text-center bg-white py-2">
 							<p class="font-weight-bold text--size-13 mb-0">Ticket Price</p>
-							<p class="mb-0 text--size-13">{{printf "%0.2f" .StakeInfo.Difficulty}}&nbsp;DCR</p>
+							<p class="mb-0 text--size-13"><a href="https://dcrdata.decred.org/charts?chart=ticket-price&zoom=month" target="_blank" rel="noopener noreferrer">{{printf "%0.2f" .StakeInfo.Difficulty}}&nbsp;DCR</a></p>
```

Comment:
Hardcoded mainnet - deliberate?

Reply:
Also I notice that we are using `dcrdata.decred.org` here, but the link added further down is `mainnet.decred.org` .

My understanding was that we should use `dcrdata.org`, but I could be wrong about that. I would like to try and keep consistency where possible though.

Reply:
ah, no that's a mistake.  I think we've still using dcrdata.org since the two hosts (mainnet.dcrdata.org and testnet.dcrdata.org) exists, while the ones on decred.org still point to Insight.